src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -407,9 +407,7 @@ def get_refresh_token(self, resource=CLOUD.endpoints.active_directory_resource_i
         if user_type == _USER:
             _, _, token_entry = self._creds_cache.retrieve_token_for_user(
                 username_or_sp_id, account[_TENANT_ID], resource)
-            refresh_token = token_entry[_REFRESH_TOKEN] if _REFRESH_TOKEN in token_entry else None
-            access_token = token_entry[_ACCESS_TOKEN] if _ACCESS_TOKEN in token_entry else None
-            return None, refresh_token, access_token, str(account[_TENANT_ID])
+            return None, token_entry.get(_REFRESH_TOKEN), token_entry[_ACCESS_TOKEN], str(account[_TENANT_ID])
```

Comment:
This change indicates `_ACCESS_TOKEN` must exist, otherwise `KeyError` will be thrown. What changed?